packages/grafana-ui/src/components/Modal/ModalTabsHeader.tsx

```diff
@@ -8,7 +8,7 @@ interface ModalTab {
   value: string;
   label: string;
   icon?: IconName;
-  labelSuffix?: () => JSX.Element;
+  labelSuffixText?: string;
```

Comment:
This change should be done here as well: https://github.com/grafana/grafana/blob/main/public/app/features/dashboard/components/ShareModal/types.ts#L13

Reply:
It's done in this PR: https://github.com/grafana/grafana/pull/44155/files#diff-cc694fdf522af8120f4ff95184d373466a71e5ff474943cbc27e3db71faf6d8eR13